### PR TITLE
Added a link variant to the button component

### PR DESCRIPTION
**Pull request by patricklafrance**
Issue: https://github.com/gsoft-inc/sg-orbit/issues/239

## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/button.overrides [hunk 1]
```diff
@@ -8,7 +8,7 @@
 .ui.button,
 .ui.buttons .button {
     min-height: @scaleHotel;
-    display: flex;
+    display: inline-flex;
```
Comment (patricklafrance) [inline packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/button.overrides]: What do you think @fraincs ? With `display: flex` the button itself was not taking 100% of it's parent, but it's container was.

File: packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/button.overrides [hunk 2]
```diff
@@ -1020,6 +1019,34 @@
...
+    box-shadow: none !important;
+}
+
+/* SKIN | Link | Basic | STATE | Active */
+.ui.button.link.basic:focus,
+.ui.button.link.basic:hover,
+.ui.button.link.basic:active {
+    text-decoration: none;
+    color: var(--primary-600) !important;
+}
```
Comment (patricklafrance) [inline packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/button.overrides]: What do you think @fraincs ?

File: packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/button.overrides (hunk 1 above)
Comment (fraincs) [reply packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/button.overrides]: all good that's the use of inline-flex

File: packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/button.overrides [hunk 3]
```diff
@@ -1020,6 +1019,34 @@
...
+    background: none !important;
+    color: var(--primary-600);
+    font-weight: 400;
+}
+
+/* SKIN | Link | STATE | Active */
+.ui.button.link:focus,
+.ui.button.link:hover,
+.ui.button.link:active {
+    text-decoration: underline;
```
Comment (fraincs) [inline packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/button.overrides]: In the best world we would not use      text-decoration: underline;  but a border bottom

**Pull request merged**